README.md

```diff
@@ -20,4 +20,4 @@ Add your answers to this markdown file.
 
 2. The time compelexity could only be expected to be log(10000) if the tree is balanced, but there is no way to know the exact time it would take to find the same element. It would depend on many different factors such as hardware and the data that was put into the tree. 
 
-3. One reason this could have happened is because the tree is unbalanced, which causes the computer to do more work in search of the element. Another reason may be that the size of the array is too big for one memory allocation, so the computer has to use a larger but slower memory unit to store the tree. A third reason this could happen is because the search function for the element in the tree is written enefficiently and calls unnecessary function calls or iterations. 
+3. One reason this could have happened is because the tree is unbalanced, which causes the computer to do more work in search of the element. Another reason may be that the size of the array is too big for one memory allocation, so the computer has to use a larger but slower memory unit to store the tree. A third reason this could happen is because the search function for the element in the tree is written enefficiently and calls unnecessary function calls or iterations..
```

Comment:
Could an unbalanced tree account for a difference that large? And for both runs you'd have to fetch the elements from main memory.

Reply:
If the tree was unbalanced to the point where its a linked list, couldnt that have a big effect on the time?

Reply:
You know how to judge that :)
That would make it linear, so what time would you expect then?